generator/src/main/java/com/scottlogic/deg/generator/generation/fieldvaluesources/TemporalFieldValueSource.java

```diff
@@ -15,7 +15,7 @@
 
 public class TemporalFieldValueSource implements FieldValueSource {
 
-    public static final LocalDateTime ISO_MAX_DATE = LocalDateTime.of(9999, 12, 31, 23, 59, 59, 999);
+    public static final LocalDateTime ISO_MAX_DATE = LocalDateTime.of(9999, 12, 31, 23, 59, 59, 999_000_000);
```

Comment:
need to make sure this doesn't revert the change from PR #723

Reply:
Done